TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Can we slim this .csproj file down to the bare necessities?  That way can easily gauge what "hacks" are left in.

For example CodeAnalysisRuleSet, FileAlignment, Prefer32Bit seem like they can all be removed.